x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engines/components/tables/engines_table.tsx

```diff
@@ -98,12 +98,18 @@ export const EnginesListTable: React.FC<EnginesListTableProps> = ({
           data-telemetry-id="entSearchContent-engines-table-viewEngineIndices"
           onClick={() => viewEngineIndices(engine.name)}
         >
-          <FormattedMessage
-            id="xpack.enterpriseSearch.content.enginesList.table.column.view.indices"
-            defaultMessage="{indicesLength} indices"
-            values={{
-              indicesLength: indices.length,
-            }}
+          <FormattedNumber value={indices.length} />
+          &nbsp;
+          <FormattedPlural
```

Comment:
Is there a reason to **not** use this? I did a search and we don't use it anywhere else in our plugin, or all of Kibana, which I was surprised by.

Reply:
oh, I like that better.